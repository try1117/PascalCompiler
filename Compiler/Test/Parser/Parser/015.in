program test;
var
  a : integer = 5;
const
  b : double = a - 2.0;
var
  c : double = a + b;

begin
end.
