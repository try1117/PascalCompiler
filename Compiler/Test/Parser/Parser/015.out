test : function()
   resultType : Nil

test declarations:
   a : Const Integer
   |-- 5

   b : Const Double
   |-- 3.000000

   c : Double
   |-- 8.000000

